app/app.go

```diff
@@ -128,6 +128,7 @@ func NewSgnApp(logger tlog.Logger, db dbm.DB, baseAppOptions ...func(*bam.BaseAp
 	}
 	viper.SetDefault(common.FlagStartMonitor, true)
 	viper.SetDefault(common.FlagEthPollInterval, 5)
+	viper.SetDefault(common.FlagEthConfirmCount, 7)
```

Comment:
what is this?

Reply:
confirmation count for safe block

Reply:
Please add TODO to this and this EthPollInterval above to move these parameters to the config later.